.github/workflows/build.yml

```diff
@@ -342,7 +330,7 @@ jobs:
         gh release create latest \
           --prerelease \
           --title "HashLink Nightly Build" \
-          "artifact/hashlink-${short_commit}-darwin.tar.gz#hashlink-latest-darwin.tar.gz" \
-          "artifact/hashlink-${short_commit}-linux-amd64.tar.gz#hashlink-latest-linux-amd64.tar.gz" \
-          "artifact/hashlink-${short_commit}-win32.zip#hashlink-latest-win32.zip" \
-          "artifact/hashlink-${short_commit}-win64.zip#hashlink-latest-win64.zip"
+          "darwin-make-64/hashlink-${short_commit}-darwin.tar.gz#hashlink-latest-darwin.tar.gz" \
+          "linux-make-64/hashlink-${short_commit}-linux-amd64.tar.gz#hashlink-latest-linux-amd64.tar.gz" \
+          "windows-vs2019-32/hashlink-${short_commit}-win32.zip#hashlink-latest-win32.zip" \
+          "windows-vs2019-64/hashlink-${short_commit}-win64.zip#hashlink-latest-win64.zip"
```

Comment:
Why are we changing this?

Reply:
In v4 of actions/upload-artifact CI artifacts have become immutable, so each matrix combination now uploads a uniquely named artifact instead of adding the zip/tar to the existing artifact.
The artifacts used for the release are still the same as before.